modules/admin_manual/pages/configuration/server/app_commands/activity_commands.adoc

```diff
@@ -0,0 +1,18 @@
+= Activity 
+:page-partial:
+
+The `activity` commands are used for automating activity notifications in ownCloud server.
+
+----
+ activity
+  activity:send-emails  Send all pending activity emails now
+----
+
+== activity:send-emails
+
+The `activity:send-emails` command sends all pending activity emails as soon as possible.
```

Comment:
not 'as soon as possible' but immediately regardless the time they are scheduled